common/scripts/lib/_helpers.sh

```diff
@@ -6,6 +6,7 @@
 # TODO: break up this file into smaller, logically-grouped
 # files after we add release and re-install features
 
+source "$LIB_DIR/_logger.sh"
```

Comment:
why is this needed?

Reply:
removed the source 
able to run installer correctly 
```
# 14:08:44 #######################################
# Booting database
##################################################
|___ Running database container
|___ Running a new database container
29264c9558140e0cd3e9f39bfb22d41a1f29673d4aa74d4a14bac6ecd6290aa7
|___ Checking db container status on: localhost:5432
|___ Waiting for db to start
|___ db found
|___ Database container successfully running
```